js/input.js

```diff
@@ -0,0 +1,13 @@
+const input = document.querySelector("#input");
+const error = document.querySelector("#input-error");
+
+input.addEventListener("keyup", function (e) {
+  const { value } = e.target;
+  console.log({ e });
+  if (value.length > 4) {
```

Comment:
When merging do remove all the console log

Reply:
Thanks for pointing.